CHANGES

```diff
@@ -1,6 +1,12 @@
 PySurfer Changes
 ================
 
+Version 0.8
+-----------
+
+- The surface geometry that is displayed can now be changed after initializing
+  a ``Brain`` instance with e.g. ``Brain.set_surf("smoothwm")``.
```

Comment:
A tiny thing but it should probably by `brain.set_surf("smoothwm")` as it's an instance method.